### PR TITLE
Update 2.3.0

### PR DESCRIPTION
**Pull request by redkale**

### CODE REVIEW

File: frameworks/Java/redkale/src/main/java/org/redkalex/benchmark/Service.java [hunk 1]
```diff
@@ -34,8 +39,8 @@
     }
 
     @RestMapping(name = "json")
-    public Message getHelloMessage() {
-        return new Message("Hello, World!");
+    public Object getHelloMessage() {
+        return JSON_TEST ? msgjson : new Message("Hello, World!");
```
Comment (nbrady-techempower) [inline frameworks/Java/redkale/src/main/java/org/redkalex/benchmark/Service.java:43]: Please revert this. This function should always instantiate a new object. Code for testing something different shouldn't be committed here.

Comment (redkale) [reply frameworks/Java/redkale/src/main/java/org/redkalex/benchmark/Service.java:43]: This is just a debugging code. The reason why the test data used to locate JSON is not high depends on whether it is IO problem or JSON parsing problem

Comment (redkale) [reply frameworks/Java/redkale/src/main/java/org/redkalex/benchmark/Service.java:43]: The next update will delete this code

Comment (nbrady-techempower) [reply frameworks/Java/redkale/src/main/java/org/redkalex/benchmark/Service.java:43]: Unfortunately I’m not going to merge debugging code that breaks the test rules. It’s too easy to forget about that with the number of tests here. You’ll have to test these types of things locally.

Comment (redkale) [reply frameworks/Java/redkale/src/main/java/org/redkalex/benchmark/Service.java:43]: OK, I'll take it out now

Comment (nbrady-techempower) [reply frameworks/Java/redkale/src/main/java/org/redkalex/benchmark/Service.java]: Thanks!

**Pull request merged**